bundles/core/src/main/java/com/adobe/cq/wcm/core/components/commons/link/Link.java

```diff
@@ -84,6 +86,17 @@ default String getProcessedURL() {
         return null;
     }
 
+
+    /**
+     * The full URL which also contains the scheme and host information.
+     * This is usually created with a {@link com.day.cq.commons.Externalizer} service
+     * @return Full link URL or {@code null} if link is invalid or can't be externalized.
+     */
+    @Nullable
+    default String getFullURL() {
```

Comment:
I would rather we stick to the following names: `getURL`, `getMappedURL` and `getExternalizedURL`.

Reply:
We may want to introduce `getCustomURL` as an extension point to allow customized processing specific to a project.